OpenOversight/app/utils.py

```diff
@@ -45,6 +45,10 @@
 )
 
 
+# HTTP Method constants
+HTTP_METHOD_GET = "GET"
+HTTP_METHOD_POST = "POST"
```

Comment:
These method constants come standard in python 3.11, but we aren't there yet. Whenever we get there, we can remove these.

Reply:
Cool, thanks for calling that out here